docs/start/concepts/application.rst

```diff
@@ -0,0 +1,75 @@
+.. _concept_application:
+
+Application
+===========
+
+An *Application* represents a single deployable application in Shipper. It
+resides in a single namespace. If this object does not exist for a given user
```

Comment:
I believe it might be clarified a bit more: application object or namespace object?